chromium_src/chrome/browser/download/download_ui_context_menu.h

```diff
@@ -6,19 +6,29 @@
 #ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_DOWNLOAD_DOWNLOAD_UI_CONTEXT_MENU_H_
 #define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_DOWNLOAD_DOWNLOAD_UI_CONTEXT_MENU_H_
 
+#include "brave/browser/download/brave_download_commands.h"
+#include "chrome/browser/download/download_commands.h"
+#include "chrome/browser/download/download_item_model.h"
+
 // Scrubs out code for histogram.
 #define RecordCommandsEnabled(...)      \
   RecordCommandsEnabled(__VA_ARGS__) {} \
   void RecordCommandsEnabled_Unused(__VA_ARGS__)
 
+// Make GetMenuModel() virtual so that we can override it in
+// DownloadUIContextMenuView for Brave.
 #define GetMenuModel \
   UnUsed() {         \
     return nullptr;  \
   }                  \
   virtual ui::SimpleMenuModel* GetMenuModel
 
+// Replace DownloadCommands with BraveDownloadCommands in the member variable.
```

Comment:
Comments should explain the "why", the "what" here is self-evident. It's not clear why this is needed here when it wasn't before

Reply:
It's still not clear why we need this now when we didn't before since there was already a reference to one here https://github.com/brave/brave-core/pull/32211/files#diff-ea50bb3f2723cfb3d7dc4893e2baa2e3bb4e8767a9074ca389b5b473acbe34f2L37